pkg/tuned/tuned.go

```diff
@@ -1044,21 +1103,27 @@ func (c *Controller) changeWatcher() (err error) {
 			if fsEvent.Op&fsnotify.Write == fsnotify.Write {
 				klog.V(1).Infof("write event on: %s", fsEvent.Name)
 				c.change.bootcmdline = true
+				c.wqTuneD.AddRateLimited(wqKey{kind: wqKindDaemon})
 			}
 
 		case err := <-wFs.Errors:
 			return fmt.Errorf("error watching filesystem: %v", err)
 
-		case <-tickerUpdate.C:
-			klog.V(2).Infof("tickerUpdate.C")
+		case <-c.changeCh:
+			var synced bool
+			klog.V(2).Infof("changeCh")
 			if c.daemon.reloading {
-				// Do not reload the Tuned daemon unless it finished reloading
+				// Do not reload the TuneD daemon unless it finished with the previous reload.
+				c.changeChRet <- synced
```

Comment:
are you relying on Go default bool value for `synced [:= false]` ?
if yes, I find it quite confusing (perhaps that's a common way to proceed in Go?), 
if no then that's a mistake as `synced` isn't explicitly initialized, AFAICU

Reply:
reading the code outside of the diff, I understand better this line ... but still find it confusing, I would rather write `c.changeChRet <- false` 
and remove the `var synced bool` above

Reply:
Yes, I rely on the Go default bool value for `synced [:= false]`?  But I also agree that explicit is probably better in this case.  Will change to `c.changeChRet <- false`.